shimmingtoolbox/unwrap/prelude.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""prelude is a wrapper function to FSL Prelude
+
+
+"""
+
+import numpy as np
+import os
+import nibabel as nib
+import subprocess
+
+
+def prelude(complex_array, affine, mask=np.array([-1]), path_2_unwrapped_phase='./unwrapped_phase.nii',
```

Comment:
Normally, verbs like is are not in variable or argument names. They are reserved for function names (significance of action) and should not be used in these.

Reply:
Not sure I understand your comment.